uint64 timestamp		# time since system start (microseconds)

uint8[50] text
uint8 severity # log level (same as in the linux kernel, starting with 0)
